sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -56,13 +56,14 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
+  set(UNIFIED_RUNTIME_REPO "https://github.com/bensuo/unified-runtime.git")
+
   # commit 3487672ceba0fd3575b5f3f15a832b100dc5fbad
   # Author: Artur Gainullin <artur.gainullin@intel.com>
   # Date:   Fri Feb 16 09:59:50 2024 -0800
   #
   #     [UR] Provide flexibility to replace unified-memory-framework repo and tag
-  set(UNIFIED_RUNTIME_TAG 3487672ceba0fd3575b5f3f15a832b100dc5fbad)
+  set(UNIFIED_RUNTIME_TAG cmdbuf-support-hip)
```

Comment:
Taking the PR out of draft to avoid bottle-necking UR merge pipelines, but UR PR needs merged first and this PR shouldn't be merged until this REPO/TAG is updated to that merged commit.